### PR TITLE
[11.0][FIX] stock_move_location: Not check availability when do immediate transfer.

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa

### CODE REVIEW

File: stock_move_location/wizard/stock_move_location.py [hunk 1]
```diff
@@ -130,8 +130,6 @@ def action_move_location(self):
         picking = self._create_picking()
         self._create_moves(picking)
         if not self.env.context.get("planned"):
-            picking.action_confirm()
-            picking.action_assign()
```
Comment (pedrobaeza) [inline stock_move_location/wizard/stock_move_location.py:134]: I think assign action is still needed

Comment (sergio-teruel) [reply stock_move_location/wizard/stock_move_location.py:134]: Why?

Comment (pedrobaeza) [reply stock_move_location/wizard/stock_move_location.py:134]: Because being immediate transfer doesn't grant you that you have enough available stock.

Comment (sergio-teruel) [reply stock_move_location/wizard/stock_move_location.py:134]: If you re-check availability the lots informed in wizard lines can be recreate with other ones...
In wizar lines already check availability quantity.
https://github.com/OCA/stock-logistics-warehouse/blob/c9af30a89e6efb339655a1d2a79edcc0c39c6a88/stock_move_location/wizard/stock_move_location_line.py#L110

Comment (pedrobaeza) [reply stock_move_location/wizard/stock_move_location.py:134]: OK, got it. Thanks.

**Review by pedrobaeza — APPROVED** [review]

**Comment by rousseldenis** [comment]
@pedrobaeza Do you know why runbot does not want to build ?

**Comment by sergio-teruel** [comment]
I do not know, lately runbot fails a lot....

**Comment by rousseldenis** [comment]
@gurneyalex

**Review by rousseldenis — APPROVED** [review]
Code review.

Fast-tracking one

**Pull request merged**